server/services/apiService.js

```diff
@@ -0,0 +1,213 @@
+const redditFetcher = require('./redditService');
+const databaseService = require('./database/databaseService');
+const constants = require('../constants');
+const {
+    SUBREDDITS_TABLE_TITLE,
+    POSTS_TABLE_TITLE,
+    STATUS_CODE_OK,
+    STATUS_CODE_FAIL,
+    ERROR_MESSAGE_SUBREDDIT_NOT_FOUND,
+    ERROR_MESSAGE_SUBREDDIT_ALREADY_EXISTS,
+    ERROR_MESSAGE_POSTS_NOT_FOUND,
+    COMMENTS_TABLE_TITLE
+} = constants;
+
+
+function init (app) {
```

Comment:
Add linter to server too?

Reply:
fixed https://github.com/YuriyBC/Reddit-archive/commit/bf46801a8ca486bcb87a245038f6c118b09092dd